src/now.rs

```diff
@@ -31,6 +42,15 @@ mod sys {
         feature = "tzdb-concatenated"
     ))]
     pub(crate) fn monotonic_time() -> Option<std::time::Instant> {
+        // See above for rationale behind this panic.
+        #[cfg(all(
+            not(feature = "js"),
+            any(target_arch = "wasm32", target_arch = "wasm64"),
+            target_os = "unknown"
+        ))]
+        panic!("getting the current time in wasm32-unknown-unknown is not possible, \
+                enable jiff's `js` feature if you are targeting a browser environment");
```

Comment:
I think for here, you can just return `None` instead for this `cfg`. This is only used internally for caching, and Jiff can deal (perhaps sub-optimally) with environments that don't provide monotonic time.